Updated Jenkins plugins - fixes asterisk issue

jcpowermac opened the pull request:
Updating credentials-binding-plugin that resolves
[JENKINS_41760](https://issues.jenkins-ci.org/browse/JENKINS-41760)

codificat commented on jenkins/plugins.txt:
```diff
@@ -1,2 +1,5 @@
-# If we need to monitor memory usage
-monitoring:1.67.0
+credentials-binding:1.13
```
Should we add a comment explaining why we need this?  i.e. point to the Jira issue... and maybe briefly mention why the others are here too?

codificat reviewed (approved):
Just a comment/question/suggestion but otherwise LGTM

jcpowermac committed:
Updated Jenkins plugins - fixes asterisk issue

Updating credentials-binding-plugin that resolves
[JENKINS_41760](https://issues.jenkins-ci.org/browse/JENKINS-41760)

jcpowermac commented:
@codificat added comments.
